charts/fhir-gateway/templates/gateway-deployment.yaml

```diff
@@ -78,6 +81,10 @@ spec:
                 secretKeyRef:
                   name: {{ include "fhir-gateway.db-secret-name" . }}
                   key: {{ include "fhir-gateway.db-secret-key" . }}
+            {{- else }}
```

Comment:
Muss hier noch `SPRING_DATASOURCE_URL=""` gesetzt werden?

Reply:
Müsste nicht weil es der Default ist in der applicationyaml. Allerdings wäre es vielleicht tatsächlich schöner es explizit auf "" zu setzen nochmal.